cfme/tests/infrastructure/test_config_management.py

```diff
@@ -24,11 +25,25 @@ def config_system(config_manager):
     return fauxfactory.gen_choice(config_manager.systems)
 
 
+@pytest.fixture
+def ansible_tower_dialog_through_ui(appliance):
+    """Returns service dialog object."""
+    service_dialogs = appliance.collections.service_dialogs
+    service_dialog = service_dialogs.instantiate(
```

Comment:
Q: why are instantiate dialog without creating it? I think we need to create it here.